system/jlib/jsocket.cpp

```diff
@@ -4130,6 +4133,7 @@ class CSocketSelectThread: public CSocketBaseThread
         dummysockopen = false;
         opendummy();
         terminating = false;
+        finished = false;
```

Comment:
This should be in initialised in the base class (if it is required)

Reply:
Done.